solanaj/src/main/java/org/p2p/solanaj/kits/transaction/network/transaction/TransactionInformationResponse.kt

```diff
@@ -0,0 +1,13 @@
+package org.p2p.solanaj.kits.transaction.network.transaction
+
+import com.google.gson.annotations.SerializedName
+
+data class TransactionInformationResponse(
+    @SerializedName("message")
+    val message: MessageResponse,
+
+    @SerializedName("signatures")
+    val signatures: List<String>
+) {
+    fun getTransactionId(): String = signatures.first()
```

Comment:
are we sure that signatures will not be empty ?

Reply:
ofc not, but if they are empty - it's an exception for us really